mucommander-translator/src/main/resources/dictionary.properties

```diff
@@ -566,6 +566,10 @@ file_viewer.close = Close
 file_viewer.large_file_warning = This file may be too large for this operation.
 file_viewer.open_anyway = Open anyway
 file_viewer.fullscreen = Full screen
+file_viewer.window_position_x = Window position X
+file_viewer.window_position_y = Window position Y
+file_viewer.window_width = Window width
+file_viewer.window_height = Window height
```

Comment:
where would these values be presented?

Reply:
I followed "template" from TextViewerPreferences, but it's probably not needed...